bin/makepdf

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+# Use and adopt this script only if there are multiple manuals
+# Each manual build is based on a book template found in books. It is mandatory to be present
+# This script creates pdf versions of the manual
+# If no manual is in the parameter list, all available manuals are built to pdf
+# If called from the command line, the script user the term "master" as reference
```

Comment:
```suggestion
# If called from the command line, the script uses the term "master" as reference
```

And we might want to stop using the term "master" where possible.

Reply:
For political correctness reasons I assume? Should be easy here, but that raises the question if we should rename our master branches to 'main'.

Reply:
I raised issue https://github.com/owncloud/enterprise/issues/4560 for discussing about that.
I just thought that we may as well not add more uses of the word "master".
Example of the industry discussion: https://thenewstack.io/words-matter-finally-tech-looks-at-removing-exclusionary-language/